src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -54,6 +54,11 @@ public static bool IsLegacyPlaceholderTemplateSite(this IEnvironment environment
             return string.IsNullOrEmpty(siteName) ? false : siteName.Equals(ScriptConstants.LegacyPlaceholderTemplateSiteName, StringComparison.InvariantCultureIgnoreCase);
         }
 
+        public static bool IsHttpInvokerEnabled(this IEnvironment environment)
+        {
+            return environment.GetEnvironmentVariable(FunctionsHttpInvoker) == "1";
```

Comment:
Nit: do you mind adding a TODO with linked work item if this is temporary?

Reply:
will do!

Reply:
removed in this part 2 PR.